docs/installation.rst

```diff
@@ -1,36 +1,46 @@
 Installation
 ============
 
-Prerequisites:
+Prerequisites: - Python 3.5, 3.6 or 3.7 - pip - Tensorflow 2.0, if pip
+errors that it can’t find it for your python/pip version
 
-* Python 2.7, 3.5 or 3.6
-* pip
+Installing all dependencies in separate conda environment:
 
-Installing all dependencies in sparate conda environment:
+.. code:: sh
 
-``conda env create -f environment.yml``
+   conda env create -f environment.yml
 
-``source activate mcfly``
+   # activate this new environment
+   source activate mcfly
 
 To install the package, run in the project directory:
 
 ``pip install .``
 
 Installing on Windows
----------------------
-When installing on Windows, there are a few things to take into consideration. The preferred (in other words: easiest) way to install Keras and mcfly is as follows:
-
-* Use `Anaconda <https://www.continuum.io/downloads>`_
-* Use Python 3.x, because tensorflow is not available on Windows for Python 2.7
-* Install numpy and scipy through the conda package manager (and not with pip)
-* To install mcfly, run `pip install mcfly` in the cmd prompt.
-* Loading and saving models can give problems on Windows, see https://github.com/NLeSC/mcfly-tutorial/issues/17
+~~~~~~~~~~~~~~~~~~~~~
+
+When installing on Windows, there are a few things to take into
+consideration. The preferred (in other words: easiest) way to install
+Keras and mcfly is as follows: \* Use
+`Anaconda <https://www.continuum.io/downloads>`__ \* Use Python 3.x,
+because tensorflow is not available on Windows for Python 2.7 \* Install
```

Comment:
This comment on Python 2.7 seems outdated... shouldn't we remove it?

Reply:
yes, I copied this from the README so I will remove it there as well.

Reply:
Great, thanks!